op-e2e/gas_price_oracle_test.go

```diff
@@ -47,14 +94,28 @@ func TestGasPriceOracle(t *testing.T) {
 			return err
 		}
 
-		used, err := caller.GetL1GasUsed(&bind.CallOpts{}, b)
+		used, err := caller.GetL1Fee(&bind.CallOpts{}, b)
 		if err != nil {
 			return err
 		}
 
-		expected := (types.FlzCompressLen(b) + 68) * 16
-		assert.Equal(t, used.Uint64(), uint64(expected), path)
+		l1BaseFee, err := caller.L1BaseFee(&bind.CallOpts{})
 
+		if err != nil {
+			return err
+		}
+
+		l1BaseFeeScaled := uint64(baseFeeScalar) * l1BaseFee.Uint64() * 16
+		l1BlobBaseFee, err := caller.BlobBaseFee(&bind.CallOpts{})
+		if err != nil {
+			return err
+		}
+
+		l1BlobFeeScaled := uint64(blobFeeScalar) * l1BlobBaseFee.Uint64()
+		l1FeeScaled := l1BaseFeeScaled + l1BlobFeeScaled
+		fastLzLength := types.FlzCompressLen(b) + 68
```

Comment:
nit: move the `+68` into the statement below for consistency with the other `+68`